src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $day-gap;
+  max-width: ($day-size * 7) + ($day-gap * 6);
+
+  @each $day, $count in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: ($day-size + $day-gap) * $count;
```

Comment:
The `margin-left` for the first child of `.calendar` should be calculated using the `$day-gap` variable alone, not `$day-size + $day-gap`. The day size should not affect the margin between the days.